console/src/components/Services/Data/DataSubSidebar.js

```diff
@@ -120,7 +122,7 @@ class DataSubSidebar extends React.Component {
                 data-test={tableName}
               >
                 <i
-                  className={styles.tableIcon + ' fa fa-table'}
+                  className={[styles.tableIcon, 'fa', iconStyle].join(' ')}
```

Comment:
Let's use string literal for simplicity and consistency
```suggestion
                  className={`${styles.tableIcon} fa ${iconStyle}`}
```